TeamForward/client/src/pages/UpdateProfile.js

```diff
@@ -59,7 +64,51 @@ const UpdateProfile = () => {
     });
   };
 
-  function updateProfile(form) {
+  const uploadProfilePicture = async(imgFile) => {
+    if(!user.s3ProfilePhotoKey && imgFile){
+      const photoData = new FormData();
+      photoData.append('photo', imgFile);
+      console.log("update profile, post call - no userprofileKey ")
+      try{
+        const response = await axios
```

Comment:
I would move the axios calls into a separate method and return the response so you can call `const response = handlePhotoUpdate(...)` in this function instead of having to mess around with a let statement.

That will also let you consolidate down to a single try/catch